styles/style.css

```diff
@@ -159,12 +196,40 @@ p {
 }
 
 
-/* Article */
+.headingdown {
+  margin : auto 0;
+  padding-top: 2.5em;
+
+}
 
 .article {
   margin-bottom: 1rem;
+  display: flex;
+  flex-direction: row;
+  border: solid 0.1em rgb(240, 226, 226);
+  padding: 1em;
+  
+  
+  }
+  
+  
+.container {
+    display: flex;
+    margin: 0px;  
+    padding-top: 2em;
+    
+} 
+
+
+.box {
+  display: flex;
+  flex-wrap: wrap;
+  float: left;
```

Comment:
Don't use `float` here you don't need it. Flexbox was created to replace `float` as an easier way to manage layouts. 
What other flexbox rules have you learned that could help with aligning this box to the left?

Reply:
removed float